src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -77,13 +110,14 @@ struct TvgSwEngine : TvgEngineMethod
 struct TvgWgEngine : TvgEngineMethod
 {
     #ifdef THORVG_WG_RASTER_SUPPORT
-        WGPUInstance instance{};
         WGPUSurface surface{};
     #endif
 
     ~TvgWgEngine()
     {
         #ifdef THORVG_WG_RASTER_SUPPORT
+            wgpuDeviceRelease(device);
```

Comment:
Hmm... this is a bad. initialize them in global init()  and terminate them by an instance...
A better is to init() - term() pairing.

Reply:
@hermet Agree, instance/adapter/device should be released in global scope like `init()`. I'll leave surfaceRelease here and rearrange the others in new `term()` function.

Reply:
Updated for this, please have a look. In binding, we'll need to call `init()` - `term()` at pair